src/test/tfs_rest_parser_row_test.cpp

```diff
@@ -109,6 +116,10 @@ TEST(TFSRestParserRow, ParseValid2Inputs) {
                                                               //============
                                                               11.0, 12, 13.0,
                                                               14.0, 15.0, 16.0));
+
+    char expectedBinary[] = {57, 28, 52};
+    EXPECT_EQ(inputC.string_val()[0], std::string(&expectedBinary[0], &expectedBinary[0] + 3));
```

Comment:
```suggestion
    EXPECT_EQ(inputC.string_val()[0], std::string(expectedBinary, expectedBinary + 3));
```
Isnt it sufficient?

Reply:
That's correct, I'll change that in all places